pagopa/io-app
# chore: [IOBP-530] Removed CTA button in payment confirmation screen with one PSP element

## ⚠️ This PR depends on https://github.com/pagopa/io-app-design-system/pull/185 ⚠️

## Short description
This PR removes the CTA button and label if only one PSP is present for the selected payment method.

## List of changes proposed in this pull request
- Upgraded the design system library to have the updated `ModuleCheckout` with the possibility of not having a CTA.
- Renders the ctaText only if the pspList has more than one element

## How to test
- Start the new payment flow from the developer playground
- Edit the `io-dev-app-server` to have only one PSP from the list inside the `src/features/wallet/payloads/transactions.ts` file
- Check that the `Edit` CTA is not present while reaching the payment confirmation screen

## Preview
|Before|After|
|-|-|
|<img src="https://github.com/pagopa/io-app/assets/34343582/899fda96-ec6a-439f-b9ea-d64691071938" width="300" />|<img src="https://github.com/pagopa/io-app/assets/34343582/5b0d58d5-9d41-45dd-9517-7a6f5a11c785" width="300" />|
